src/components/UI/MovieCategoryDropdown.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="dropdown-categories">
+    <label class="dropdown-categories__label" for="">Categories</label>
+    <select
+      class="dropdown-categories__select"
+      @change="$store.commit('setSelectedGenre', $event.target.value)"
+    >
+      <option
+        class="dropdown-categories__options"
+        v-for="genre in ['All Categories', ...$store.getters.genreList]"
+        :key="genre"
+      >
+        {{ genre }}
+      </option>
+    </select>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "MovieCategoryDropdown",
+  props: {
+    listOptions: {
+      type: Array,
+      default: null,
+    },
+  },
```

Comment:
I think we do not use this prop at the moment - the options are accessed from the state instead of a prop so we can safely remove this 😄
```suggestion
```